packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -78,7 +79,8 @@ export async function importBlock(
   const currentEpoch = computeEpochAtSlot(currentSlot);
   const blockEpoch = computeEpochAtSlot(blockSlot);
   const prevFinalizedEpoch = this.forkChoice.getFinalizedCheckpoint().epoch;
-  const blockDelaySec = (fullyVerifiedBlock.seenTimestampSec - postState.genesisTime) % this.config.SECONDS_PER_SLOT;
+  const blockDelaySec =
+    fullyVerifiedBlock.seenTimestampSec - computeTimeAtSlot(this.config, blockSlot, postState.genesisTime);
```

Comment:
noticed this changes behavior slightly but it's a bug fix as previously we would apply proposer boost to a block that's more than 12 seconds late but before 4 seconds in next slot because we didn't calculate delay based on block slot